pages/organizations/_organizationId/contacts/_contactId/index.vue

```diff
@@ -362,13 +373,21 @@ export default {
       organization   : {},
       inputBoxId     : 0,
       titleOptions   : [ ...TITLE_OPTIONS ],
+      actions        : [
+        {
+          text: 'Save', type: 'primary', name: 'save', default: true,
+        },
+        { text: "Don't save", type: 'danger', name: 'next' },
+        { text: 'Cancel', name: 'cancel' },
```

Comment:
align code. make them all one line or all expanded
```suggestion
        { name: 'save',   text: 'Save',       type: 'primary',  default: true },
        { name: 'next'    text: "Don't save", type: 'danger' },
        { name: 'cancel', text: 'Cancel' },
```

Reply:
DONE! changes done as suggested.